docs/en/advanced-concepts/wda-custom-server.md

```diff
@@ -28,6 +28,12 @@ cd /usr/local/lib/node_modules/appium/node_modules/appium-xcuitest-driver/WebDri
 ./Scripts/bootstrap.sh -d
 ```
 
+Also, it might be necessary to create an empty folder for WDA resources:
+
+```bash
+mkdir -p /usr/local/lib/node_modules/appium/node_modules/appium-xcuitest-driver/WebDriverAgent/Resources/WebDriverAgent.bundle
```

Comment:
We do, if they're allowing Appium to automate the process. But we can't if they are managing things themselves.